scripts/get-examples.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
How do you feel about naming it `scripts/examples/index.sh`? I'm not asking you to change this: I'd be happy to make a PR for renaming all the scripts.